### DISCUSSION TITLE
Correct way to link to root?

### DISCUSSION THREAD

**Discussion by tomcam** [community/community#23674] [Other Feature Feedback, Questions, & Ideas]
<p>Is this the right way to link back to the root README.md of a project?</p>
<p><code>[Back](/README.md)</code></p>
<p>Other methods such as these seem to fail:</p>
<pre><code class="lang-auto">[Back](/)
[Back](/.)
</code></pre>

**Comment by otter-computer** [comment, marked as answer]
<p>There isn’t really a “correct” way to link back to the root, but usually the more explicit the better. Both of these examples should work though:</p>
<pre><code class="lang-markdown">[Back](/)
[Back](/README.md)
</code></pre>
<p>Be aware that the first link won’t work when viewing the rendered file on <a href="http://GitHub.com">GitHub.com</a> in some instances, just down to the way GitHub constructs repository URLs compared to your Pages site. The second link should work for both though.</p>
<p>The following should work, assuming the file that contains these links is inside a single directory:</p>
<pre><code class="lang-markdown">[Back](../)
[Back](../README.md)
</code></pre>

**Reply by Castillon35** [reply]
Help me to get successfully root access
